scripts/ci/run-jenkins.sh

```diff
@@ -74,6 +74,12 @@ if [ -x "/usr/bin/dpkg-architecture" ];
 	wget -qO- https://download.mono-project.com/test/new-certs.tgz| tar zx -C ~/.config/.mono/
 fi
 
+if [[ ${CI_TAGS} == *'win-'* ]];
```

Comment:
@akoeplinger is this correct way ?

Reply:
we tried it, not sure if it works (the tests don't run on Windows in the current version of the PR because of the path separator issue).